jlm/llvm/opt/alias-analyses/Worklist.hpp

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright 2024 Håvard Krogstie <krogstie.havard@gmail.com>
+ * See COPYING for terms of redistribution.
+ */
+#ifndef JLM_LLVM_OPT_ALIAS_ANALYSES_WORKLIST_HPP
+#define JLM_LLVM_OPT_ALIAS_ANALYSES_WORKLIST_HPP
```

Comment:
Could we move the entire file to to `util`? These implementations are useful for other parts in the compiler as well.